src/test/java/bio/terra/service/load/LoadUnitTest.java

```diff
@@ -0,0 +1,94 @@
+package bio.terra.service.load;
+
+import bio.terra.common.category.Unit;
+import bio.terra.service.load.exception.LoadLockFailureException;
+import bio.terra.service.load.exception.LoadLockedException;
+import bio.terra.service.load.flight.LoadMapKeys;
+import bio.terra.stairway.FlightContext;
+import bio.terra.stairway.FlightMap;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.startsWith;
+import static org.junit.Assert.assertThat;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+@AutoConfigureMockMvc
+@Category(Unit.class)
+public class LoadUnitTest {
+    private static final String tag1 = "myLoadTag1";
+    private static final String tag2 = "myLoadTag2";
+    private static final String flight1 = "myFlightId1";
+    private static final String flight2 = "myFlightId2";
+
+    @Autowired
+    private LoadService loadService;
+
```

Comment:
Any good way to add a concurrent-lock test here?

Reply:
Not here, but we could make a connected test and use fault insertion to get two flights  past `load.isLocked() == false` and make sure that one of them fails with the right exception.